docs/releases/2.2.txt

```diff
@@ -435,6 +435,16 @@ Miscellaneous
 * :attr:`Group.name <django.contrib.auth.models.Group.name>` ``max_length``
   is increased from 80 to 150 characters.
 
+* The :class:`django.test.Client` class now raises a ``TypeError`` if ``None``
```

Comment:
I think the notes could be combined in some fashion to reduce redundancy. e.g. "Client / urlencode now raise TypeError if None is passed as a value of the data argument to the request methods / the query argument, respectively.

Maybe you want to try something a bit different.